policies/recipes/artifacts.go

```diff
@@ -34,7 +35,8 @@ import (
 func fetchArtifacts(ctx context.Context, artifacts []*agentendpointpb.SoftwareRecipe_Artifact, directory string) (map[string]string, error) {
 	localNames := make(map[string]string)
 
-	for _, a := range artifacts {
+	for i, a := range artifacts {
+		logger.Debugf("Downloading artifact %d: %q", i, a)
```

Comment:
np: i don't think the index of the artifact matters at all, since they're all named